includes/mailpoet/class-main.php

```diff
@@ -91,7 +91,7 @@ public function __construct( $mailpoet_api ) {
 	 * @return Sensei_MailPoet
 	 */
 	public static function get_instance( $mailpoet_api = null ) {
-		if ( is_null( $mailpoet_api ) ) {
+		if ( is_null( $mailpoet_api ) && class_exists( \MailPoet\API\API::class ) ) {
```

Comment:
In this case we return null from this method.
Not sure if the client code expects that...